src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/RolesApiTest.java

```diff
@@ -144,7 +144,7 @@ public void testRolesApi() throws Exception {
 
         // hidden role
         response = rh.executeGetRequest("/_opendistro/_security/api/roles/opendistro_security_internal", new Header[0]);
-        Assert.assertEquals(HttpStatus.SC_NOT_FOUND, response.getStatusCode());
+        Assert.assertEquals(HttpStatus.SC_OK, response.getStatusCode());
```

Comment:
For "OK" response, should it also check the content of the response?

Reply:
we can check. I will add a few

Reply:
We can add. Adding a few now

Reply:
added